### PR TITLE
Feature/show price on icon

### PR DESCRIPTION
**Pull request by ghost**
#### What is the purpose of this pull request?
Added a new prop `showPrice`to the minicart. If this prop is `true`then the total price of the items in the cart will appear beside the car if there are items in the cart.

#### What problem is this solving?
https://github.com/vtex-apps/store-discussion/issues/41

#### How should this be manually tested?
Initialize the minicart with the prop `showPrice` as true, then add an item with a price greater than 0 to the cart and watch the price being displayed at the side of the cart.

#### Screenshots or example usage
image

image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [2.22.0] - 2019-07-16
```
Comment (klzns) [inline CHANGELOG.md]: Do not remove previous versions. Changelogs are only incremental

File: react/index.js [hunk 2]
```diff
@@ -302,12 +309,14 @@ class MiniCart extends Component {
...
       showShippingCost,
       showTotalItemsQty,
+      showPrice,
       linkState: { minicartItems: items, orderForm, isOpen },
     } = this.props
 
     const itemsToShow = this.getFilteredItems()
     const totalItemsSum = arr => arr.reduce((sum, product) => sum + product.quantity, 0)
     const quantity = showTotalItemsQty ? totalItemsSum(itemsToShow) : itemsToShow.length
+    const totalPrice = this.sumItemsPrice(itemsToShow)
```
Comment (klzns) [inline react/index.js]: I think we should just get the Total value from orderForm instead of computing this "by hand"

File: react/index.js [hunk 3]
```diff
@@ -366,9 +382,21 @@ class MiniCart extends Component {
                   </span>
                 )}
               </span>
-              {iconLabel && (
-                <span className={iconLabelClasses}>{iconLabel}</span>
-              )}
+              <span className="flex flex-column items-start">
```
Comment (klzns) [inline react/index.js:464]: Wrap this whole thing in a condition to only create this element if `iconLabel || (showPrice && totalPrice > 0)`.

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/index.js (hunk 3 above)
Comment (lbebber) [reply react/index.js:464]: and perhaps create something like `const isPriceVisible = showPrice && totalPrice > 0`, and use it both here and down below

**Review by klzns — APPROVED** [review]

**Pull request merged**